src/tools/tidy/src/ui_tests.rs

```diff
@@ -18,7 +18,7 @@ const ENTRY_LIMIT: usize = 900;
 // FIXME: The following limits should be reduced eventually.
 
 const ISSUES_ENTRY_LIMIT: usize = 1733;
-const ROOT_ENTRY_LIMIT: usize = 860;
+const ROOT_ENTRY_LIMIT: usize = 859;
```

Comment:
No longer necessary.
```suggestion
const ROOT_ENTRY_LIMIT: usize = 860;
```